assets/js/util.js

```diff
@@ -308,17 +332,21 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(useAjax) {
+        return $.ajax(requestOptions);
```

Comment:
Um, using `GET` does pass the request arguments. I am using `GET` because modifying it to make it usable with `jsonp()` would be ideal. I am not sure what other parameters count while considering behaviour?

a. With GET:

![screenshot from 2017-08-01 00-09-23](https://user-images.githubusercontent.com/8894636/28792750-3d5e59bc-764e-11e7-9033-0a5ad4dd00b7.png)

b. With POST:

![screenshot from 2017-08-01 00-11-52](https://user-images.githubusercontent.com/8894636/28792758-442cd6ba-764e-11e7-8e1d-efae979e6458.png)